tests/utils/utils.ts

```diff
@@ -222,23 +222,20 @@ function getConfig(ruleName: string, inputFile: string) {
   const filename = inputFile.slice(inputFile.indexOf(ruleName))
   const code = fs.readFileSync(inputFile, "utf8")
   let config
-  let configFile: string = inputFile.replace(/input\.svelte$/u, "config.json")
+  let configFile: string = inputFile.replace(/input\.[a-z]+$/u, "config.json")
   if (!fs.existsSync(configFile)) {
     configFile = path.join(path.dirname(inputFile), "_config.json")
   }
   if (fs.existsSync(configFile)) {
     config = JSON.parse(fs.readFileSync(configFile, "utf8"))
   }
+  const parser =
+    path.extname(filename) === ".svelte"
+      ? require.resolve("svelte-eslint-parser")
+      : undefined
   if (config && typeof config === "object") {
```

Comment:
Is this check necessary? Shouldn't `return Object.assign({ parser }, config, { code, filename })` always work?

Reply:
Thank you for let me know. I didn't notice it. I fixed it.